nntrainer/optimizers/lr_scheduler.h

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file   lr_scheduler.h
+ * @date   09 December 2021
+ * @brief  This is Learning Rate Scheduler interface class
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#ifndef __LEARNING_RATE_SCHEDULER__
+#define __LEARNING_RATE_SCHEDULER__
+#ifdef __cplusplus
+
+#include <string>
+
+#include <node_exporter.h>
```

Comment:
Could you put this inside cpp? this is *.hpp and quite heavy.

we will need to fwd declare like below

https://github.com/nnstreamer/nntrainer/blob/e333b7ffe402eb3572b3082f0bfc8b103ba58a4e/nntrainer/optimizers/optimizer_devel.h#L27-L28

Reply:
done.